python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -63,8 +64,11 @@ def from_config_value(inst_data, config_value):
         return EcsRunLauncher(inst_data=inst_data, **config_value)
 
     def _set_ecs_tags(self, run_id, task_arn):
-        tags = [{"key": "dagster/run_id", "value": run_id}]
-        self.ecs.tag_resource(resourceArn=task_arn, tags=tags)
+        try:
+            tags = [{"key": "dagster/run_id", "value": run_id}]
+            self.ecs.tag_resource(resourceArn=task_arn, tags=tags)
+        except ClientError:
```

Comment:
Maybe we also want to log that we're not tagging with directions to upgrade to the new long ARN format? Does that strike us as an engine event? Something we should instead just spit out as a warning in a normal logger?

Reply:
My gut is it shouldn't go in the event log (the impact of internal ECS resources not getting logs seems fairly low for a relative edge case), but normal logger warning seems OK?

Reply:
I'm actually just going to punt on logging to get this out the door.